<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <property name="windowIcon">
   <iconset theme="weather"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#0697da;
</string>
  </property>
  <widget class="QLabel" name="temperature">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="weatherlabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Partly Sunny</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string> Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="sunrise">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string> Sunrise/Sunset</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>375</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="wind">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Wind</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="query">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;
border-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>  Search Location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="queryLabel">
      <property name="font">
       <font>
        <family>Yu Gothic UI</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="queryButton">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;
border-color: rgba(1,148,217,255);
background-color: rgba(1,148,217,255);</string>
      </property>
      <property name="text">
       <string> Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <widget class="QLabel" name="rise">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;</string>
      </property>
      <property name="text">
       <string>  Sunrise : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sunriseValue">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>00:00          </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>591</width>
     <height>83</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="location">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;</string>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="set">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;</string>
      </property>
      <property name="text">
       <string>  Sunset : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sunsetValue">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>00:00          </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QLabel" name="windSpeed">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;</string>
      </property>
      <property name="text">
       <string>  Wind Speed : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="windSpeedValue">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>    0 m/s  </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="windDir">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ffffff;</string>
      </property>
      <property name="text">
       <string>  Wind Direction : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="windDirValue">
      <property name="font">
       <font>
        <family>Open Sans Semibold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(170, 255, 255);</string>
      </property>
      <property name="text">
       <string>North   </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="celciusTemp">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="text">
    <string>°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="pressurevalue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>134</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);
</string>
   </property>
   <property name="text">
    <string>0 hPa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="humidityValue">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);
</string>
   </property>
   <property name="text">
    <string>  0%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pressure">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Pressure</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="humidity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans Semibold</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;</string>
   </property>
   <property name="text">
    <string>Humidity</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>500</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>500</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 255, 255);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
